### PR TITLE
net-im/gitter-bin: new ebuild

### PR DESCRIPTION
**Pull request by SpiderX**
Gitter is a chat and networking platform that helps to manage,
grow and connect communities through messaging, content and discovery.

Gitter is designed to make community messaging, collaboration and discovery
as smooth and simple as possible. You can easily create, organise and
grow your communities, inviting others to join just in one click.

Package-Manager: Portage-2.3.3, Repoman-2.3.1

### CODE REVIEW

**Comment by monsieurp** [comment]
Hi. Thanks for your contribution. I will give it a go and get back to you.

**Comment by SpiderX** [comment]
Hello. Are there some issues with this package?

File: net-im/gitter-bin/gitter-bin-3.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+inherit eutils gnome2-utils unpacker
+
+DESCRIPTION="Where developers come to talk"
+HOMEPAGE="http://www.gitter.com/"
+SRC_URI="
+	amd64? ( https://update.gitter.im/linux64/${MY_PN}_${PV}_amd64.deb )
+	x86? ( https://update.gitter.im/linux32/${MY_PN}_${PV}_i386.deb )"
+LICENSE="no-source-code"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```
Comment (gktrk) [inline net-im/gitter-bin/gitter-bin-3.1.0.ebuild]: You might want to consider excluding other arches explicitly by using `-*`

File: net-im/gitter-bin/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>spiderx@spiderx.dp.ua</email>
+		<name>Vladimir Pavljuchenkov</name>
+		<description>Proxy maintainer</description>
```
Comment (gktrk) [inline net-im/gitter-bin/metadata.xml]: Don't need the description. Also, we'd call you proxied-maintainer, not proxy-maintainer ;)

File: net-im/gitter-bin/gitter-bin-3.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit eutils gnome2-utils unpacker
+
+DESCRIPTION="Where developers come to talk"
```
Comment (SpiderX) [inline net-im/gitter-bin/gitter-bin-3.1.0.ebuild]: I modified it to make it more clear

File: net-im/gitter-bin/gitter-bin-3.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+HOMEPAGE="http://www.gitter.com/"
```
Comment (SpiderX) [inline net-im/gitter-bin/gitter-bin-3.1.0.ebuild]: Missed this. Thanks.

File: net-im/gitter-bin/gitter-bin-3.1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+SRC_URI="
+	amd64? ( https://update.gitter.im/linux64/${MY_PN}_${PV}_amd64.deb )
+	x86? ( https://update.gitter.im/linux32/${MY_PN}_${PV}_i386.deb )"
+LICENSE="no-source-code"
```
Comment (SpiderX) [inline net-im/gitter-bin/gitter-bin-3.1.0.ebuild]: I have not found any license in portage tree that can be used.
Did you mean that I have to submit a license for gitter?
If yes should this be done in another PR?

File: net-im/gitter-bin/gitter-bin-3.1.0.ebuild (hunk 1 above)
Comment (SpiderX) [inline net-im/gitter-bin/gitter-bin-3.1.0.ebuild]: Added.

File: net-im/gitter-bin/metadata.xml (hunk 2 above)
Comment (SpiderX) [inline net-im/gitter-bin/metadata.xml]: Removed. Changed.

**Comment by SpiderX** [comment]
Should I submit a license for gitter based on its ToS and create a new PR for this?

**Comment by gktrk** [comment]
Sorry for the delay. You don't need a new PR, just add the ToS under `licenses/gitter` and update this PR.

**Comment by SpiderX** [comment]
Done. Please, review.

**Comment by gktrk** [comment]
Merged in https://github.com/gentoo/gentoo/compare/6dcba13...d1e45d6. Thanks!
